apache/superset
# chore: Add Applitools logo to the docs homepage

### SUMMARY
Adds the Applitools logo and URL in the docs homepage

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

![Screenshot 2022-07-29 at 18 19 42](https://user-images.githubusercontent.com/60598000/181791736-b763761a-f46c-49f5-a457-636a62c3595b.png)


### TESTING INSTRUCTIONS
1. Open the docs
2. Make sure the Applitools logo is showing at the bottom
3. Make sure it works well in different screen sizes

### ADDITIONAL INFORMATION
